src/components/select/simple-select/simple-select.stories.mdx

```diff
@@ -199,9 +200,11 @@ The inline label can change to be top aligned at a breakpoint. Enable this by pa
   </Story>
 </Preview>
 
-## Props:
-
-<Props of={ Select } />
+<StyledSystemProps
+  of={ Select }
+  spacing
+  spacingDefaults={{ mt: '3', mb: '3' }}
```

Comment:
The `select` component does not seem to have the `mt` and `mb` default props listed here.

Reply:
![image](https://user-images.githubusercontent.com/14963680/93986324-1c41c380-fd7e-11ea-9cfe-97ef4fcc0675.png)

Were you looking at the experimental select rather than the Design System one?

Reply:
I meant that the component, when viewed in the sandbox doesn't have a margin-top or margin-bottom of 3 by default.

Reply:
If it did have those defaults this test should be failing 
```js
 testStyledSystemSpacing(props => <SimpleSelect { ...props } />);
```